src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -4,11 +4,11 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Vrednost bi trebalo da bude netačna.</target>
+                <target>Vrednost bi trebala da bude netačna.</target>
```

Comment:
You are right, this is the correct form for this verb when subject is preceding it. Could you change every occurence of it, where applicable?

Reply:
@ajakov sure commit coming :)